### DISCUSSION TITLE
需要增加移动端的登录方法

### DISCUSSION THREAD

**Discussion by KoldeC** [zhaohongxuan/obsidian-weread-plugin#119] [Q&A]
需要增加仅用移动端即可而不用使用桌面系统的登录方法。

**Comment by zhaohongxuan** [comment]
这个要看后面obisidian的API了，现在是不行，因为登录依赖desktop的组件。

**Comment by hellohellochatgpt** [comment]
> 这个要看后面obisidian的API了，现在是不行，因为登录依赖desktop的组件。

两年过去了，不知道支持了没有呢
